docs/products/clickhouse/concepts/disaster-recovery.rst

```diff
@@ -0,0 +1,65 @@
+Disaster recovery in Aiven for ClickHouse®
+==========================================
+
+Disaster recovery is a process of coping with emergencies or crises using dedicated methods for protecting resources and/ or reestablishing their desired status. In the context of data infrastructure, well-established disaster recovery methods are of a particular importance for preventing data loss or corruption. Software failure, loss of an availability zone, or datacenter outage are only a few examples of emergencies when disaster recovery comes in. This article helps you understand how Aiven for ClickHouse® prevents and mitigates such emergencies and what disaster recovery methods it uses to keep your data safe and sound.
+
+High availability
+-----------------
+
+High availability (HA) is an entity's ability to continuously maintain a certain level of operational performance for a desired period of time. HA is typically achieved by redundancy - securing replicas of databases or services to be highly available. To support disaster recovery technologies, a database service needs to stay highly available, for example, by operating on a few nodes holding the same data.
+
+With Aiven, HA for your service is supported in business and premium plans. See `Plan comparison <https://aiven.io/pricing?tab=plan-comparison&product=clickhouse>`_ for details.
+
+.. seealso::
+
+    :ref:`Cross-availability-zone data distribution <cross-zone data distro>` 
+
+Backup and restore
+------------------
+
+Backups
+'''''''
+
+Backups of Aiven for ClickHouse services happen daily and contain database lists, table schemas, table content, and access entities (such as users or roles). Aiven for ClickHouse supports incremental backups: files already present in the object storage are not re-uploaded and only changed parts are backed up.
+
+Recovery
+''''''''
+
+The restoration of a backup of an Aiven for ClickHouse service is performed on a running ClickHouse server and proceeds as a regular power-on of the service. The restoration happens only for powering up a service after powering it down or forking a service.
+
+.. seealso::
+
+    For more information on backups in Aiven, see :doc:`Backups at Aiven </docs/platform/concepts/service_backups>`.
+
+Sharding
+--------
+
+Essentially, sharding is a technique of splitting database rows across multiple database nodes, which usually significantly increases performance. However, integrating sharding with database replication technologies, data can be replicated across shards of the sharded database. Replication at the shard level provides high availability and helps to achieve disaster recovery. A shard group can be replicated to one or more data centers, which improves the disaster recovery capability.
+
```

Comment:
I think here we need an extra paragraph to explicitly define the specifics of sharding/replication in Aiven for ClickHouse, something like:

"With business and premium plans, each shard is replicated across 3 availability zones. The service and the data stays fully available even if an entire availability zone is lost".

I think it's worth it because the paragraph above and the linked reference about availability zones only say "can" and do not describe exactly what is promised in Aiven for ClikHouse and sharded services.
(We _could_ be less careful and not properly spread each shard over multiple availability zones for instance.)